Artwork Endpoint

Micalobia opened the pull request:
Added the artwork endpoint, as well as support for thumbsize to work on fake_image_url, if it's there.

KevinPayravi commented on app.py:
```diff
@@ -319,9 +319,13 @@ def call_cargo(parameters, request_args): # Request args are passed in just for
                 except:
                     abort(500, description=error_response("Error while getting image CDN thumbnail URL.", "Failure occured with the following parameters: {}.".format(parameters)))
-
+                
```
Small lint issue, extra spaces on this new line

Micalobia commented:
Okay, so I've done the following:.
- Removed the `image` field from art.
- Added the `_pageName=url` field to art.
- Changed the `fake` field in art to `has_fake`.
- Changed the `?fake=bool` query in the `/nh/art` endpoint to `?hasfake-bool`.
- Renamed the `get_art` and `get_art_all` functions to `get_nh_art` and `get_nh_art_all` to comply with similar functions.
- Fixed the linting issue on line 328.

Micalobia commented:
Forgot docs; On that now

Micalobia commented:
Should be good to go, long as I didn't miss anything

KevinPayravi reviewed (approved):
LGTM

KevinPayravi commented:
Great job setting this up, and thanks for the contribution!
